### DISCUSSION TITLE
Unable to run the ttest

### DISCUSSION THREAD

**Discussion by dipalietggs** [Xceptance/XLT#512] [General]
When I am running the test it does not show the result. In the background, the server is running. 
![image](https://github.com/user-attachments/assets/051796ec-3217-44ff-b83a-db8f931a754b)

**Comment by js-xc** [comment]
Can you please add more details what you did so we can help you? 
You should be able to run any of the test classes (the java files in the `test` folder) as a JUnit test. While the test runs, data and errors are shown in the console/terminal of your IDE, and when the test is finished there should be a link to the [result browser](https://docs.xceptance.com/xlt/load-testing/manual/440-result-browser/). In your screenshot I only see some project build result messages?

**Reply by dipalietggs** [reply]
So When I am trying to run the test it shows no test found in the selected file or folder you can see in the left bottom corner. I have Junit downloaded in my VS code. But when I am keeping Junit dependencies in the pom.xml file it shows the error. Not sure why i am not able to run the test classes
![image](https://github.com/user-attachments/assets/a3878ecb-5af6-4229-8467-b2ba04333780)

**Reply by js-xc** [reply]
It should not be necessary to add the junit dependency to the pom.xml - `posters-simple-loadtest-suite` should work out of the box, just as downloaded from github. There might be a few additional steps needed as you are using VS code, please [check the documentation](https://docs.xceptance.com/xlt/load-testing/how-tos/vs-code-test-run/) to learn more.
